bugsnag/src/main/java/com/bugsnag/BugsnagAppender.java

```diff
@@ -524,10 +524,10 @@ public void setAppVersion(String appVersion) {
      * Internal use only
      * Should only be used via the logback.xml file
      *
-     * @param metaData Adds meta data to every report
+     * @param metadata Adds metadata to every report
      */
-    public void setMetaData(LogbackMetaData metaData) {
-        this.globalMetaData.add(metaData);
+    public void setMetadata(LogbackMetadata metadata) {
+        this.globalMetadata.add(metadata);
     }
```

Comment:
I wonder whether it's not worth having a deprecated delegate here:

```java
@Deprecated
public void setMetaData(LogbackMetaData metaData) {
    setMetadata(metaData);
}
```

That way code using this method will be pushed to move, but logging config won't break. @tomlongridge thoughts?